examples/lighting-app/silabs/SiWx917/src/main.cpp

```diff
@@ -75,6 +78,9 @@ int main(void)
         appError(CHIP_ERROR_INTERNAL);
     }
 
+    //Change the button flag to enable button interrupts
+	ButtonFlag = 1;
```

Comment:
What is the use of the ButtonFlag? It is set to 1 at init and then never changes. The condition in sl_button_on_change is then always true

Reply:
it is a bug in HW , we are trying to figure out the cause.
Button interrupt were automatically been generated at boot up time continuously and it is calling the Handler which interrupts the boot process.

So to avoid this I added this flag so to handler immediately terminates, it is temporary fix.

Reply:
Do we have a ticket to track this issue?